### ISSUE TITLE
[UI] Deploy an application from Github

### ISSUE THREAD

**Issue by richard-cox** [epinio/ui#149]
- Covers step 1 from https://github.com/epinio/ui/issues/63#issuecomment-1164180440
- User can select `Github` as a source and is guided through org/user --> repo --> branch to select a commit

**Comment by mmartin24** [comment]
Checked on latest epinio and in general seem to be working fine. I can create an app from an specific commit using the Gihub source type

![image](https://user-images.githubusercontent.com/37271841/185954295-2bb2104d-f37e-4d2b-b6eb-d9cbd3bba04c.png)
![image](https://user-images.githubusercontent.com/37271841/185954550-70cea702-aedb-43ed-a5e2-f632cc78a338.png)


Nevertheless I opened this issue because it seems possible to bypass this flow by returning to the previous screen: https://github.com/epinio/ui/issues/153 . I am not sure if this wants to be treated separately or as part of this ticket. For the time being I just opened separately.
